helm-cli.js

```diff
@@ -274,11 +300,14 @@ function splitDirectory(directory) {
   ];
 }
 
-// Helm Docker image accepts commands WITHOUT the CLI name
-function sanitizeCommand(command) {
-  return command.startsWith(HELM_CLI_NAME)
-    ? command.slice(HELM_CLI_NAME.length).trim()
-    : command;
+function extractChartDirectoryFromCommand(command) {
+  const directory = command.match(ABSOLUTE_PATH_REGEXP);
```

Comment:
There exists a 'extractPathsFromCommand` function in the kaholo-plugin-library for the path extraction, it covers a broad range of possible path arguments in a command.

Reply:
Thanks, I forgot about that one.